src/runtime.cc

```diff
@@ -267,6 +241,33 @@ namespace detail {
 
 	host_object_manager& runtime::get_host_object_manager() const { return *m_host_object_mngr; }
 
+	std::string runtime::gather_command_graph() const {
+		const auto graph_str = m_schdlr->print_command_graph();
+
+		// Send local graph to rank 0
+		if(m_local_nid != 0) {
+			const uint64_t size = graph_str.size();
```

Comment:
The size field can be an `int` because we can only send `int`-sized payload anyway.

Reply:
Done!